src/lando/lando/settings.py

```diff
@@ -10,24 +10,41 @@
 https://docs.djangoproject.com/en/dev/ref/settings/
 """
 
+
+import os
 from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
 
+def get_setting(
```

Comment:
We should support type hints for these settings in some way. We could create methods with concrete return types like `get_string_setting` etc, or we could add type hints to each setting definition, like `SECRET_KEY: str = get_setting` etc below. Using the `target_type` parameter is fine if we go with type hints on the definition, but without adding hints the LSP can't infer the type.

Reply:
Out of the two things you suggested, I think the latter is better. This particular implementation to get settings is very preliminary, mainly to support being able to not track secrets which I am using to connect to Auth0. I think we will have to create something more robust as a follow up regardless.

Reply:
After fixing up the `.env` in `compose.yaml`, this method is actually no longer needed since Docker can pick up the environment variables from the `.env` file directly. Will remove this, but we may have to revisit this in the future.